src/graphql/schema/User/resolvers.js

```diff
@@ -36,6 +37,30 @@ export default {
         throw new Error('User could not be created!');
       }
     },
+    facebook: async (_, { email, accessToken, userID, user = {} }, { User }) => {
+      //this should be an enviromental variable but will be kept here for now for checking
+      const serverAccessToken = '295087824459443|ierZVxTgM2AUv0aBDR4j2jCLeno'
+      const uri = `https://graph.facebook.com/debug_token?input_token=${accessToken}&access_token=${serverAccessToken}`;
+      let verification = await fetch (uri);
+      let verificationResponse = await verification.json();
+      if (verificationResponse.data.is_valid && verificationResponse.data.user_id === userID) {
+        const user = await User.findOne({ email });
+        console.log(user);
+        if (user) return getJWT({ _id: user._id, email: user.email });
+        console.log('here');
+        try {
+          const currentUser = await User.findOneAndUpdate({ email }, { email, ...user }, { upsert: true });
```

Comment:
What is the User document being updated with here?  I'm don't understand what the update is for since none of the updated parameters seems to have changed since the db read on line 47.